### ISSUE TITLE
"Noone" isn't a word

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2543]
|[<img alt="bitglue's avatar" src="https://avatars.githubusercontent.com/u/1232472?s=50" width="50" height="50">](https://github.com/bitglue)| @bitglue reported|
|-|-|
|Trac ID|trac#2543|
|Type|defect|
|Created|2007-03-29 15:29:59Z|

http://twistedmatrix.com/projects/core/documentation/howto/udp.html says 'print "Noone listening"'

Attachments:

* [noone.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/413/4133aae6eb4eb007c49649c9acb27fb7a3d13476/d1005c3d84d6d03fbe270002ac2d29748b2542ec.diff) (2016 bytes) - added by indigo on 2007-05-08 14:52:48Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__2543 2543
type__defect defect
reporter__indigo indigo
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1175182199000000 1175182199000000
changetime__1183059561000000 1183059561000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph set owner to @bitglue|
|-|-|

Doc patches would help bump the priority here.

**Comment by twisted-trac** [comment]
|[<img alt="bitglue's avatar" src="https://avatars.githubusercontent.com/u/1232472?s=50" width="50" height="50">](https://github.com/bitglue)<a name="note_2"></a>|@bitglue set owner to @glyph|
|-|-|

boink

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve set owner to @therve|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set status to `closed`|
|-|-|

(In [20653]) Remove typo from udp documentation.

Author: indigo
Reviewer: therve
Fixes [#2543](https://github.com/twisted/twisted/issues/2543)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
